xwiki-platform-core/xwiki-platform-websocket/src/main/java/org/xwiki/websocket/script/WebSocketScriptService.java

```diff
@@ -84,9 +84,13 @@ public String url(String pathOrRoleHint)
             XWikiContext xcontext = this.xcontextProvider.get();
 
             StringBuilder path = new StringBuilder("/");
-            path.append(xcontext.getWiki().getWebAppPath(xcontext));
-            path.append("websocket").append(path(pathOrRoleHint));
 
+            // Test if installed with short URL and avoid double slash
```

Comment:
This comment could be improve by mentioning XWiki installed under the ROOT servlet context (since Short URL is something different)